bin.src/ingestPredefinedFluxCal.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python3
+import argparse
+import os
+import typing
+
+from lsst.daf.butler import Butler, DatasetRef, DatasetType, DimensionGroup, FileDataset
+from lsst.resources import ResourcePath
+
+
+def main() -> None:
+    """Ingest a predefined fluxCal.
+
+    This program registers a single fluxCal many times
+    with various visit numbers
+    so that the pipeline will use it to calibrate all these visits.
+    """
+    parser = argparse.ArgumentParser(
+        description=typing.cast(str, main.__doc__),
+    )
+    parser.add_argument(
+        "fluxCal",
+        help="""
+        Path to a predefined fluxCal
+        """,
+    )
+    parser.add_argument(
+        "-b",
+        "--butler-config",
+        metavar="URI",
```

Comment:
Are the upper-case `metavar` values necessary? It looks like they get translated to lower-case, so why not provide lower-case originally so the relation is clearer?